### PR TITLE
docs: add turkish readme file

### PR DESCRIPTION
**Pull request by basalumutgazi**
Readme page's Turkish translation has been added.

### CODE REVIEW

File: lang/tr/README.md [hunk 1]
```diff
@@ -0,0 +1,243 @@
+---
+layout: page
+title: archipelago 🐚 🌊 🐬 🏄 🌴 🥥 🩴 🏖️ 👙
+permalink: /
+---
+
+[English](README.md)
+[Spanish](lang/es/README.md)
+
```
Comment (jbampton) [inline lang/tr/README.md:9]: Missing language button for Turkish

**Review by jbampton — CHANGES_REQUESTED** [review]
You need to remove all the `ALL-CONTRIBUTORS` sections since this only goes on the English README

**Comment by jbampton** [comment]
The markdownlint is also failing

File: lang/tr/README.md (hunk 1 above)
Comment (basalumutgazi) [reply lang/tr/README.md:9]: I thought we should add the Turkish button to other languages. Are you sure that you want me to add it?

File: lang/tr/README.md [hunk 2]
```diff
@@ -0,0 +1,214 @@
+---
+layout: page
+title: archipelago 🐚 🌊 🐬 🏄 🌴 🥥 🩴 🏖️ 👙
+permalink: /
+---
+
+[English](README.md)
+[Spanish](lang/es/README.md)
```
Comment (jbampton) [inline lang/tr/README.md:8]: Looks like you are missing the button for Arabic

File: lang/tr/README.md [hunk 3]
```diff
@@ -0,0 +1,215 @@
...
+[Arabic](lang/ar/README.md)
```
Comment (jbampton) [inline lang/tr/README.md:9]:
```suggestion
[Arabic](lang/ar/README.md)
[Spanish](lang/es/README.md)
```

**Pull request merged**